pkg/autoupdate/tuf/library_manager.go

```diff
@@ -138,7 +138,11 @@ func (ulm *updateLibraryManager) stageAndVerifyUpdate(binary autoupdatableBinary
 	if err != nil {
 		return "", fmt.Errorf("could not create file at %s: %w", stagedUpdatePath, err)
 	}
-	defer out.Close()
+	defer func() {
+		if err := out.Close(); err != nil {
```

Comment:
I think returning the error is more correct. If the file can't close, it means it's corrupt on disk.

Reply:
Will update in next PR

Reply:
Addressed in https://github.com/kolide/launcher/pull/1119